core/src/components/input/test/input.spec.ts

```diff
@@ -7,8 +8,32 @@ it('should inherit attributes', async () => {
     html: '<ion-input title="my title" tabindex="-1" data-form-type="password"></ion-input>',
   });
 
-  const nativeEl = page.body.querySelector('ion-input input');
+  const nativeEl = page.body.querySelector('ion-input input')!;
   expect(nativeEl.getAttribute('title')).toBe('my title');
   expect(nativeEl.getAttribute('tabindex')).toBe('-1');
   expect(nativeEl.getAttribute('data-form-type')).toBe('password');
 });
+
+describe('input masking', () => {
```

Comment:
Are these tests necessary? We're just checking if Stencil passed the correct default value here. I can't imagine the defaults changing by accident/without being caught in review.

Reply:
I don't know if any tests are necessary, but it helps avoid the potential for human error in both development and review and allows projects to iterate quicker with extra validation that existing behavior is maintained.

Reply:
I don't mind keeping them, but I will note that any change to the default value of props will cause `api.txt` to be updated so we should be able to catch accidental changes to the default value during review.